webpack.config.js

```diff
@@ -25,6 +25,8 @@ module.exports = {
     },
     entry: {
         donationFormBlock: srcPath('DonationForms/Blocks/DonationFormBlock/resources/block.ts'),
+        donationFormBlockApp: srcPath('DonationForms/Blocks/DonationFormBlock/resources/app/index.tsx'),
+        donationFormBlockAppStyle: srcPath('DonationForms/Blocks/DonationFormBlock/resources/app/styles/index.scss'),
```

Comment:
When you import a .scss file into a .tsx, wp-scripts already compile it with the same name as the compiled .js file.
In this case, you'll see a `donationFormBlockApp.css` file with the same content as this `donationFormBlockAppStyle.css`

Reply:
Ahh ok got it! That makes sense. I re-updated the wp_enqueue_style. Resolved: 0900da3f6ff61389a8c577b1445d6f2b0af98301